.stylelintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "stylelint-config-standard-scss",
+  "rules": {
+    "no-descending-specificity": null
+  },
+  "ignoreFiles": [
+    "src/**/scss/plugins/*.scss",
+    "src/Modules/Backend/assets/Backend/webpack/scss/components/_editor.scss",
```

Comment:
Why is this file ignored?

Reply:
This contains the css for ckeditor and the selectors don't match the stylelint rules